[2025-08-03T10:56:01.789+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:55:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:56:01.790+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:55:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:55:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:56:01.867+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:55:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:56:01.869+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:55:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:55:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:56:01.931+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:55:00+00:00 [queued]>
[2025-08-03T10:56:01.944+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:55:00+00:00 [queued]>
[2025-08-03T10:56:01.945+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-08-03T10:56:01.965+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_training_data> on 2025-08-03 10:55:00+00:00
[2025-08-03T10:56:01.976+0000] {standard_task_runner.py:57} INFO - Started process 1112 to run task
[2025-08-03T10:56:01.982+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'fetch_training_data', 'scheduled__2025-08-03T10:55:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmpljmi8p5g']
[2025-08-03T10:56:01.986+0000] {standard_task_runner.py:85} INFO - Job 100: Subtask fetch_training_data
[2025-08-03T10:56:02.060+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:55:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:56:02.061+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:55:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:55:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:56:02.064+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:55:00+00:00 [running]> on host 407839faddcf
[2025-08-03T10:56:02.191+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='fetch_training_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-03T10:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-03T10:55:00+00:00'
[2025-08-03T10:56:02.202+0000] {db_utils.py:17} INFO - Connected to PostgreSQL
[2025-08-03T10:56:03.340+0000] {train_predict_to_db_dag.py:29} INFO - Fetched training data tail:
            id  store  ...  productname                 created_at
100666  156106     60  ...    product_A 2025-08-03 10:55:15.175414
100667  156107     59  ...    product_A 2025-08-03 10:55:25.180894
100668  156108     58  ...    product_A 2025-08-03 10:55:35.191427
100669  156109     57  ...    product_A 2025-08-03 10:55:45.028905
100670  156110     56  ...    product_A 2025-08-03 10:55:55.045536

[5 rows x 12 columns]
[2025-08-03T10:56:03.407+0000] {python.py:194} INFO - Done. Returned value was: {'store': 370, 'start_date': '2025-05-05', 'end_date': '2025-08-03'}
[2025-08-03T10:56:03.456+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=fetch_training_data, execution_date=20250803T105500, start_date=20250803T105601, end_date=20250803T105603
[2025-08-03T10:56:03.524+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-03T10:56:03.559+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
